lib/methods/export.js

```diff
@@ -66,7 +66,7 @@ module.exports = self => {
         )
         .map(attachment => attachment._id);
 
-      const attachments = await self.getAttachments(req, attachmentsIds);
```

Comment:
req isn;t used

Reply:
but it's good apostrophe practice to have it for eventual overriding